snowpack.config.js

```diff
@@ -0,0 +1,56 @@
+const path = require('path');
+const fs = require('fs');
+const pkg = require('./package.json');
+
+const outDir = '__html';
+const exclude = new Set(['favicon.png', 'utils.js', 'template.html']);
+
+const watch = (file, cb) => {
+  cb(file);
+  return fs.watchFile(file, () => cb(file));
+};
+
+const build = (template) => {
+  const html = fs.readFileSync(template).toString();
+  if (!fs.existsSync(outDir)) {
+    fs.mkdirSync(outDir);
+  }
+  fs.readdirSync('example').forEach((file) => {
+    if (exclude.has(file)) return;
+    const outfile = path.join(outDir, file.replace('.js', '.html'));
+    fs.writeFileSync(
+      outfile,
+      html.replace(
+        '<!-- INSERT_ENTRYPOINT -->',
+        `<script type="module" src="%PUBLIC_URL%${file}"></script>`
```

Comment:
Inserts a script tag with the corresponding js entrypoint. "%PUBLIC_URL%" is resolved via snowpack config below: https://www.snowpack.dev/reference/configuration#buildoptionsbaseurl